Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -485,7 +485,9 @@ void WebProcess::initializeWebProcess(WebProcessCreationParameters&& parameters)
 
     PAL::registerNotifyCallback("com.apple.WebKit.logPageState"_s, [this] {
         for (auto& page : m_pageMap.values()) {
+#if !RELEASE_LOG_DISABLED
```

Comment:
Let’s put this `#if` around the entire `PAL::registerNotifyCallback` call and the entire lambda. There’s no need to do any of this if `RELEASE_LOG` is disabled.